pkgs/servers/web-apps/outline/default.nix

```diff
@@ -11,21 +11,21 @@
 
 stdenv.mkDerivation rec {
   pname = "outline";
-  version = "0.73.1";
+  version = "0.74.0";
 
   src = fetchFromGitHub {
     owner = "outline";
     repo = "outline";
     rev = "v${version}";
-    hash = "sha256-t1m9pKsM9E2iAg9vv/nKmQioRi6kMjFGcTXzcT3cMxs=";
+    hash = "sha256-fF//SgcBYcJmPDaev8G1s+svCW1bU9CmN3uWEoEeMUk=";
   };
 
   nativeBuildInputs = [ makeWrapper prefetch-yarn-deps ];
   buildInputs = [ yarn nodejs ];
 
   yarnOfflineCache = fetchYarnDeps {
     yarnLock = "${src}/yarn.lock";
-    hash = "sha256-TRZlkDe7ARLGmfw5a0Dw9hSBRhqWvOfuBfPZ5/Bt/TI=";
+    hash = "sha256-yrCohOgZdqrzuuaTx+exs/aPABN6Ng2Ok763L0KTWIc=";
```

Comment:
This hash seems incorrect as ofborg failes to the getting a different hash. https://logs.ofborg.org/?key=nixos/nixpkgs.275007&attempt_id=8f85beb2-b1df-4a5e-9945-109964c44d6c

Reply:
updated from the one in ofborg's output to keep moving forward, my invocation must be incorrect: `nix hash file --sri --type sha256 yarn.lock`

Reply:
That is because that hash isn't the hash of the `yarn.lock` file, but of the resulting `node-modules` folder. The only way to update this is to build it and copy the new hash from the resulting error message.